Student/Student.xcodeproj/project.pbxproj

```diff
@@ -2161,7 +2157,6 @@
 				495D8BB91CFE17D700D41098 /* LaunchScreen.storyboard in Resources */,
 				7D38A7D724CF610E00A4373E /* QuizDetailsViewController.storyboard in Resources */,
 				7DCF0FA522F8A8A9000AACCE /* QuizListViewController.storyboard in Resources */,
-				B5F122C91D8C4D6C0045063A /* Student.entitlements in Resources */,
```

Comment:
There was a warning about this not needed to get copied into the bundle, so I removed it.